File: Web/RMA-Web-Shell/projects/claimcare/src/app/claim-manager/shared/entities/claim-email.ts
import { TemplateTypeEnum } from 'projects/shared-models-lib/src/lib/enums/template-type-enum';

export class ClaimEmail {
    personEventId: number;
    claimId: number;
    templateType: TemplateTypeEnum;
    emailAddress: string;
    tokens: { [key: string]: string };
    documentId: number | null;
}

File: Service/ClaimCare/RMA.Service.ClaimCare.Contracts/Entities/ClaimEmail.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

using System.Collections.Generic;

namespace RMA.Service.ClaimCare.Contracts.Entities
{
    public class ClaimEmail
    {
        public int PersonEventId { get; set; }
        public int ClaimId { get; set; }
        public TemplateTypeEnum TemplateType { get; set; }
        public string EmailAddress { get; set; }
        public Dictionary<string, string> Tokens { get; set; }
        public int? DocumentId { get; set; }

    }
}